fendermint/app/src/cmd/run.rs

```diff
@@ -56,6 +56,8 @@ namespaces! {
 ///
 /// This method acts as our composition root.
 async fn run(settings: Settings) -> anyhow::Result<()> {
+    println!("Starting fendermint, opts: {:?}", settings);
```

Comment:
Is this line needed?

Reply:
I added this line to confirm that stdout logs were sent from fendermint to Loki, but otherwise not needed.